docs/examples/use_cases/tensorflow/yolov4/src/dali/ops.py

```diff
@@ -0,0 +1,212 @@
+import nvidia.dali as dali
+
+def input(file_root, annotations_file, device_id, num_threads, device, random_shuffle=True):
+    inputs, bboxes, classes = dali.fn.readers.coco(
+        file_root=file_root,
+        annotations_file=annotations_file,
+        ltrb=True,
+        shard_id=device_id,
+        num_shards=num_threads,
+        ratio=True,
+        random_shuffle=random_shuffle
+    )
+    images = dali.fn.decoders.image(inputs, device=device, output_type=dali.types.RGB)
+
+    return images, bboxes, classes
+
+def ltrb_to_xywh(bboxes):
+    Z = dali.types.Constant(0.0)
+    H = dali.types.Constant(0.5)
+    O = dali.types.Constant(1.0)
+    MO = dali.types.Constant(-1.0)
+
+    M = dali.fn.stack(
+            dali.fn.stack(H, Z, H, Z),
+            dali.fn.stack(Z, H, Z, H),
+            dali.fn.stack(MO, Z, O, Z),
+            dali.fn.stack(Z, MO, Z, O)
+        )
```

Comment:
```suggestion
    M = np.float32([[ 0.5,   0, 0.5,   0 ],
                    [   0, 0.5,   0, 0.5 ],
                    [  -1,   0,   1,   0 ],
                    [   0,  -1,   0,   1 ]])
```

This should do - or you can even flatten it and use a plain list, which will be passed as build-time argument:

```suggestion
    M = [ 0.5,   0, 0.5,   0,
            0, 0.5,   0, 0.5,
           -1,   0,   1,   0,
            0,  -1,   0,   1 ]
```

Reply:
Changed.